boards/shields/x_nucleo_cca02m1/doc/index.rst

```diff
@@ -0,0 +1,56 @@
+.. _x-nucleo-cca02m1:
+
+X-NUCLEO-CCA02M1: Digital MEMS microphones SHIELD for STM32 Nucleo
```

Comment:
``SHIELD`` should not be capitalized (it's not an acronym or legal name), change to ``shield``

Reply:
ack